src/main/java/uk/co/real_logic/benchmarks/rtt/aeron/AeronUtil.java

```diff
@@ -0,0 +1,249 @@
+    {
+        return getProperty(ARCHIVE_CHANNEL_PROP_NAME, IPC_CHANNEL);
+    }
+
+    static int archiveStreamId()
+    {
+        return getInteger(ARCHIVE_STREAM_ID_PROP_NAME, 1_000_000_002);
+    }
+
+    static boolean embeddedMediaDriver()
+    {
+        return getBoolean(EMBEDDED_MEDIA_DRIVER_PROP_NAME);
+    }
+
+    static int frameCountLimit()
+    {
+        return getInteger(FRAME_COUNT_LIMIT_PROP_NAME, 10);
+    }
+
+    static MediaDriver launchEmbeddedMediaDriverIfConfigured()
+    {
+        if (embeddedMediaDriver())
+        {
+            final MediaDriver.Context mediaDriverContext = new MediaDriver.Context()
+                .dirDeleteOnStart(true)
+                .spiesSimulateConnection(true);
+            return MediaDriver.launch(mediaDriverContext);
+        }
+        return null;
+    }
+
+    static ArchivingMediaDriver launchArchivingMediaDriver(final boolean recordingEventsEnabled)
+    {
+        final MediaDriver.Context mediaDriverCtx = new MediaDriver.Context()
+            .dirDeleteOnStart(true)
+            .spiesSimulateConnection(true);
+        final Archive.Context archiveCtx = new Archive.Context()
+            .aeronDirectoryName(mediaDriverCtx.aeronDirectoryName())
+            .recordingEventsEnabled(recordingEventsEnabled)
+            .deleteArchiveOnStart(true);
+        return ArchivingMediaDriver.launch(
+            mediaDriverCtx,
+            archiveCtx);
+    }
+
+    static long awaitRecordingStart(final Aeron aeron, final int publicationSessionId)
+    {
+        final CountersReader counters = aeron.countersReader();
+        int counterId;
+        do
+        {
+            counterId = findCounterIdBySession(counters, publicationSessionId);
+        }
+        while (NULL_COUNTER_ID == counterId);
+
+        return getRecordingId(counters, counterId);
+    }
+
+    static long findLastRecordingId(
+        final AeronArchive aeronArchive, final String recordingChannel, final int recordingStreamId)
+    {
+        final MutableLong lastRecordingId = new MutableLong();
+
+        final RecordingDescriptorConsumer consumer =
+            (controlSessionId,
+            correlationId,
+            recordingId,
+            startTimestamp,
+            stopTimestamp,
+            startPosition,
+            stopPosition,
+            initialTermId,
+            segmentFileLength,
+            termBufferLength,
+            mtuLength,
+            sessionId,
+            streamId,
+            strippedChannel,
+            originalChannel,
+            sourceIdentity) -> lastRecordingId.set(recordingId);
+
+        int foundCount;
+        do
+        {
+            foundCount = aeronArchive.listRecordingsForUri(0, 1, recordingChannel, recordingStreamId, consumer);
+        }
+        while (0 == foundCount);
+
+        return lastRecordingId.get();
+    }
+
+    static void pipeFragements(final Subscription from, final ExclusivePublication to, final AtomicBoolean running)
```

Comment:
@mjpt777: There are a couple of items I would like to clarify:
- Should node (server) have an `IdleStrategy` to be executed after the `poll`?
- Currently the server will exit when the client is closed due to the `Image#isClosed` check. If the check is removed then the node can remain running, i.e. one can start another benchmark without having to start server again.
- Is there an official way to avoid re-assembling the message between `poll` and `offer`? Simply passing un-assembled fragments to `offer` does not work since the resulting header will not have the `flags` field set, i.e. each fragment will be considered a separate message.

Reply:
* Server node where spinning should take a configurable `IdleStrategy`.
* Starting another benchmark without closing would be more usable.
* You can avoid re-assemble by using `tryClaim` and reading the header flags and resetting them. Let the archive use to do before `offerBlock`.

Reply:
Done